README.md

```diff
@@ -12,7 +12,14 @@ This is convenient to load data from a `backend` for instance.
 ## use
 You have to wrap your component, and give a `load` props to that resulted component.
 
-Example with `redux` :
+You can also add an optional configuration object as second parameter.
+
+Parameter | Needed | Default value | Description
+----------|--------|---------------|-------------
+`Loader` | no | `null` | A component that will be display depending on `prop` value.
+`prop` | no | `loaded` | A prop name that determine when to display the `Loader` component. The prop value should be a Boolean.
```

Comment:
I would see prop could be anything.
The `HOC` print the `Loader` component whenever the value of that prop is `false`, `undefined` or `null`.

We can make it an issue to enhance the component if you want to.